### ISSUE TITLE
Sort lobby content from master branch

### ISSUE THREAD

**Issue by SpencerSharkey** [Sass2015/Assets#1]
I've made a new branch in this repository that is rts_dev
It houses all the RTS content, and only the RTS content. It's very small, only around 22mb which is awesome. I removed deprecated models (jaanus's old ones) and only kept Mr. Giggle's.

The same needs to be done for the lounge content. "Lounge reconstructed 06" packs all it's custom textures in this addon structure, so the materials used need to be extracted and put into their own content addon. Hopefully we can cut down it's size as well.

**Comment by SpencerSharkey** [comment]
the rts_dev branch is a fully-submitted steam workshop addon, as well.
http://steamcommunity.com/sharedfiles/filedetails/?id=599432665

I have configured Sassilization gamemode to use these two content packs and auto-send them to clients via workshop. Makes bringing testers in very easy and brings us one step closer to a real server environment.

**Comment by AlexSwift** [comment]
When I buy the servers, it would be worth having the server to auto synchronize with the github assets, and upload them to the workshop. I would also suggest having a new steam account to publish under (so it's not under your name but under everyones) :P

**Comment by SpencerSharkey** [comment]
I agree, a new utility steam account would be nice. We can easily automate the github publishing with the utilities garry provides with gmod. They're pretty easy!

**Comment by JamesxX** [comment]
You could simply write a program that updates a git repo when the webhook pings it.
